src/selectors/hosts.ts

```diff
@@ -11,3 +11,19 @@ export const getHostTableRows = createSelector(
   getHosts,
   hosts => hosts.map(host => Object.values(host))
 );
+
+const hostsValuesSelector = state => state.hosts.items;
+
+export const hostsAggregateValues = createSelector(
+  hostsValuesSelector,
+  items => items.reduce((acc, item) => acc + item.memory, 0)
```

Comment:
This looks good, I assume that `acc` is going to be an object of an explicit type which holds the aggregate values. For each host, aggregate values will be increased by the values from the host.